tests/ui/if_not_else.rs

```diff
@@ -28,4 +28,46 @@ fn main() {
     } else {
         println!("Bunny");
     }
+
+    if !(foo() && bla()) {
+        #[cfg(not(debug_assertions))]
+        println!("not debug");
+        #[cfg(debug_assertions)]
+        println!("debug");
+        if foo() {
+            println!("foo");
+        } else if bla() {
+            println!("bla");
+        } else {
+            println!("both false");
+        }
+    } else {
+        println!("both true");
+    }
```

Comment:
Considering the following code, which is similar in structure to this one, except that the parts to be linted are nested:
```rust
if !(foo() && bla()) {
    if !foo() {
        println!("not foo");
    } else {
        println!("some output");
    }
} else {
    println!("both true");
}
```
Then, when I run `cargo bless`, I get a `cannot replace slice of data that was already replaced` error. But I don't think this is a problem since `rustfix` will apply the first fix and ignore the subsequent when put to practical use.